haskell/cabal
# Use Paths_cabal_install for cabal-install version number

https://github.com/haskell/cabal/issues/9415 was due to `cabalInstallVersion` having to be manually adjusted and inevitably, the package uploaded on Hackage missed that adjustment.

This PR uses `Paths_cabal_install.version` to provide a package version that is guaranteed to always be in sync with the cabal file's package version.

### QA Notes

Calling `Distribution.Client.Version.cabalInstallVersion` in ghci should return what is indicated in [cabal-install.cabal](https://github.com/haskell/cabal/blob/master/cabal-install/cabal-install.cabal#L4)
